File: main/src/main/java/net/fwitz/math/main/binary/split/functions/hyperbolic/SplitSinhZ.java
package net.fwitz.math.main.binary.split.functions.hyperbolic;

import net.fwitz.math.binary.split.SplitComplex;
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot;

public class SplitSinhZ {
    public static void main(String[] args) {
        new SplitComplexFunctionPlot("sinh(z)")
                .computeFn(SplitComplex::sinh)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/split/functions/hyperbolic/SplitSinhZ.kt
package net.fwitz.math.main.binary.split.functions.hyperbolic

import net.fwitz.math.binary.split.SplitComplex
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot

object SplitSinhZ {
    @JvmStatic
    fun main(args: Array<String>) = SplitComplexFunctionPlot("sinh(z)")
        .computeFn(SplitComplex::sinh)
        .render()
}